www/install.rst

```diff
@@ -6,13 +6,12 @@ These are instructions for installing :ref:`the full SciPy stack
 <stackspec>`.  For installing individual packages, such as NumPy and
 SciPy, see :ref:`individual-packages` below.
 
-
 Scientific Python distributions
 -------------------------------
 
-For most users, especially on Windows and Mac, the easiest way to install the
-packages of the SciPy stack is to download one of these Python distributions,
-which includes all the key packages:
+For most users, especially on Windows, the easiest way to install the packages
```

Comment:
We can do the merge, but I think we disagree on the how much better conda + infrastructure is than pip + infrastructure, so I guess we'll end up having the same discussion on the new PR.

Don't get me wrong, I'm happy to broadly recommend conda etc even though I do not use or recommend them myself.   My concern here is to remain relatively even-handed in the situation where there are trade-offs for each.